lib/generators/hobo/migration/migrator.rb

```diff
@@ -217,12 +217,18 @@ def extract_column_renames!(to_add, to_remove, table_name)
 
 
         def always_ignore_tables
-          # TODO: figure out how to do this in a sane way and be compatible with 2.2 and 2.3 - class has moved
-          sessions_table = CGI::Session::ActiveRecordStore::Session.table_name if
-            defined?(CGI::Session::ActiveRecordStore::Session) &&
-            defined?(ActionController::Base) &&
-            ActionController::Base.session_store == CGI::Session::ActiveRecordStore
-          ['schema_info', 'schema_migrations',  'simple_sesions'].compact
+          sessions_table =
+            begin
+              if defined?(CGI::Session::ActiveRecordStore::Session) &&
+                 defined?(ActionController::Base) &&
+                 ActionController::Base.session_store == CGI::Session::ActiveRecordStore
+                CGI::Session::ActiveRecordStore::Session.table_name
+              end
+            rescue
+              nil
+            end
+
+          ['schema_info', 'schema_migrations', 'ar_internal_metadata', sessions_table].compact
```

Comment:
I’m curious if there are rails variables that we could get these table names from rather than hard coding. That way if rails changes the table name between versions, we don’t need to account for it. Feel free to ignore and we can do this at another time.

Reply:
I don't know if there are...and likely if there, they probably changed across Rails versions as they refactored the code. So I'm somewhat doubtful that it would be a consistent API across versions.

Our best hope would be if Rails exposed a single method that gave this full list. That's that only way we could have kept this bug from cropping up. Maybe we create a ticket to look for that?

Reply:
Separate ticket for this sounds good after a quick look to see if it exists 😄

Reply:
TECH-4876 created.